ci/bamboo/build-manylinux-debug.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+# This script runs inside numenta's custom manylinux docker image
+# quay.io/numenta/manylinux1_x86_64_centos6 and builds the debug manylinux
+# x86_64 wide-unicode nupic.bindings wheel per PEP-513. See
+# https://github.com/numenta/manylinux.
+#
+# ASUMPTIONS: Expects a pristine nupic.core source tree without any remnant
+#             build artifacts from prior build attempts. Otherwise, behavior is
+#             undefined.
+
+
+set -o errexit
+set -o xtrace
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Configure environment for manylinux build
+source ${DIR}/manylinux-build-env.rc
```

Comment:
@scottpurdy, I used a simple way to avoid creating another layer of manylinux docker image and the corresponding repository and build system integration resources. I modeled the manylinux scripts on the existing build-linux-release.sh and its debug counterpart that both use setup-dependencies-linux.sh to set up the environment and tooling installation/configuration.

Similarly to the use of setup-dependencies-linux.sh by the existing linux build scripts, the manylinux build sources manylinux-build-env.rc to set up the necessary environment.

I think that it's a reasonable tradeoff that matches and benefits from the existing design pattern.